iothub/device/tests/Transport/Mqtt/ClientWebSocketChannelTests.cs

```diff
@@ -14,46 +14,43 @@ namespace Microsoft.Azure.Devices.Client.Test.Mqtt
     using DotNetty.Codecs.Mqtt.Packets;
     using DotNetty.Handlers.Logging;
     using DotNetty.Transport.Channels;
+    using FluentAssertions;
     using Microsoft.Azure.Devices.Client.Transport.Mqtt;
     using Microsoft.VisualStudio.TestTools.UnitTesting;
 
     [TestClass]
     [TestCategory("Unit")]
+    [DoNotParallelize]
     public class ClientWebSocketChannelTests
     {
-        const string IotHubName = "localhost";
-        const int Port = 12346;
-        static HttpListener listener;
-        static ServerWebSocketChannel serverWebSocketChannel;
-        static ReadListeningHandler serverListener;
-        static volatile bool done;
-
-        const string ClientId = "scenarioClient1";
-        const string SubscribeTopicFilter1 = "test/+";
-        const string SubscribeTopicFilter2 = "test2/#";
-        const string PublishC2STopic = "loopback/qosZero";
-        const string PublishC2SQos0Payload = "C->S, QoS 0 test.";
-        const string PublishC2SQos1Topic = "loopback2/qos/One";
-        const string PublishC2SQos1Payload = "C->S, QoS 1 test. Different data length.";
-        const string PublishS2CQos1Topic = "test2/scenarioClient1/special/qos/One";
-        const string PublishS2CQos1Payload = "S->C, QoS 1 test. Different data length #2.";
-
-        static readonly TimeSpan DefaultTimeout = TimeSpan.FromSeconds(300);
-
-        [ClassInitialize()]
-        public static void AssembyInitialize(TestContext testcontext)
+        private const string IotHubName = "localhost";
+        private const int Port = 12346;
+        private const string ClientId = "scenarioClient1";
+        private const string PublishS2CQos1Payload = "S->C, QoS 1 test. Different data length #2.";
+        private const string PublishS2CQos1Topic = "test2/scenarioClient1/special/qos/One";
+
+        private static HttpListener s_listener;
+        private static ServerWebSocketChannel s_serverWebSocketChannel;
+        private static ReadListeningHandler s_serverListener;
+        private static volatile bool s_isDone;
+
+        private static readonly TimeSpan s_defaultTimeout = TimeSpan.FromSeconds(300);
+
+        [ClassInitialize]
+        public static void ClassInitialize(TestContext testcontext)
         {
-            listener = new HttpListener();
-            listener.Prefixes.Add("http://+:" + Port + WebSocketConstants.UriSuffix + "/");
-            listener.Start();
+            s_listener = new HttpListener();
+            s_listener.Prefixes.Add("http://+:" + Port + WebSocketConstants.UriSuffix + "/");
+            s_listener.Start();
 
-            RunWebSocketServer().ContinueWith(t => t, TaskContinuationOptions.OnlyOnFaulted);
+            _ = RunWebSocketServer();
```

Comment:
We don't await this because it runs for the duration of the test, and is signalled to quit in ClassCleanup.